### PR TITLE
Price component adjustments

### PR DESCRIPTION
**Pull request by RumataDeEstor**

### CODE REVIEW

File: components/price/price.js [hunk 1]
```diff
@@ -98,6 +120,12 @@ Price.defaultProps = {
 
 Price.propTypes = {
   /**
+   * Additional text that will be displayed under price and the line.
+   * E.g.: 'per day'.
+   */
+  additionalText: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```
Comment (mAiNiNfEcTiOn) [inline components/price/price.js]: I'd add:

`PropTypes.arrayOf(PropTypes.node)`

File: components/price/price.js [hunk 2]
```diff
@@ -61,6 +63,22 @@ function Price(props) {
 
   const className = getClassNamesWithMods(rootClass, mods);
 
+  const textBlock = additionalText
+    ? (
+      <div className={`${rootClass}__additional-text-block`}>
+        {additionalText}
+      </div>
+      )
+  : null;
```
Comment (asci) [inline components/price/price.js]: indentation

File: components/price/price.js (hunk 1 above)
Comment (reaktivo) [inline components/price/price.js]: PropTypes.node covers all this cases, you can simply

```
additionalText: PropTypes.node,
```

Comment (RumataDeEstor) [inline components/price/price.js]: @reaktivo you mean for string as well?

File: components/price/price.js (hunk 2 above)
Comment (RumataDeEstor) [inline components/price/price.js]: @asci fixed, can you approve now? thx

File: components/price/price.js (hunk 1 above)
Comment (mAiNiNfEcTiOn) [inline components/price/price.js]: @reaktivo yes, you're right... didn't know that... But next time, add some factual documentation links to it: https://facebook.github.io/react/docs/typechecking-with-proptypes.html#proptypes

Doesn't hurt much 👍

**Pull request merged**